src/modules/lighthouse/showReports.ts

```diff
@@ -0,0 +1,44 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse } from '../../clients'
+
+import { TableGenerator } from './TableGenerator'
+
+function allWhenUndefined(atribute: string): string {
+  return !atribute ? '<all>' : atribute
+}
```

Comment:
Nit:
```suggestion
function allWhenUndefined(atribute: string | undefined): string {
  return atribute ?? '<all>'
}
```

Reply:
Nice! I didnt know about this notation with undefined fields